modules/Bio/EnsEMBL/Compara/RunnableDB/PairAligner/AlignmentProcessing.pm

```diff
@@ -42,6 +31,7 @@ use strict;
 use warnings;
 
 use Bio::EnsEMBL::Utils::Exception qw(throw);
+
```

Comment:
Is there a reason/rule as to why you add an additional newline after each `use package::something qw(somethingelse);`?

Reply:
I'm following the same rule as in python, dividing the `use` in groups (and leaving a newline between them):
* Standard libraries (e.g. warning, Data-Dumper, etc.)
* External libraries (e.g. Bio from BioPerl)
* EnsEMBL libraries outside Compara
* Compara libraries

Reply:
Oh, I only made a connection to the pattern of `qw()` before each. Is this something we have to do in Perl? I like the grouping/ordering, but the additional newlines hurt my eyes. Is it a standard rule in Python? Or one of the compara rules?

Reply:
It is a standard in Python, but not in Perl or Compara. I like it so that's why I do it, but I can revert it if we agree on something else.